src/main.js

```diff
@@ -58,15 +58,19 @@ if (test) {
 	document.body.removeChild(loading);
 	clearInterval(dots);
 
+	const configBlock = document.getElementById('config');
+	const pageBlock = document.getElementById('page');
```

Comment:
I think it would be slightly better to create these elements dynamically, and then append them to the `header` element.